scripts/lint-changed.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-MD_FILES=`git diff --cached --name-only | tr " " "\n" | egrep ^.*\.md$`
+MD_FILES=`git diff --cached --name-only | tr " " "\n" | grep '^.*\.md$'`
 
 # Execute Markdown lint if any markdown files have been changed and added to git
 [[ -z "$MD_FILES" ]] || GEM_PATH=.gem .gem/bin/mdl "$MD_FILES"
+exit 0
```

Comment:
Completely unnecessary. If need be, we should refactor line 6 to appropriately return an okay (no pages need changed at all, no linting required), or the output of mdl.